### PR TITLE
update package and change format string for pipelines

### PR DESCRIPTION
**Pull request by dongocanh96**

### CODE REVIEW

**Comment by dongocanh96** [comment]
@hvnsweeting Thầy check giúp em cái pull request ạ :D

File: vnw/vnw/pipelines.py [hunk 1]
```diff
@@ -29,20 +29,22 @@ def process_item(self, item, spider):
         try:
             kv = {kw: item[kw] for kw in REQUIRED_FIELDS}
         except KeyError as e:
-            logger.error('Drop job: %s %s, missing required key %r',
-                         item.get('name', 'MISSING'),
-                         item.get('url', 'MISSING'),
-                         e)
+            logger.error('Drop job: {} {}, missing required key {}'.format(
```
Comment (hvnsweeting) [inline vnw/vnw/pipelines.py]: we should keep using old format log. Or at least, don't format before passing to logger methods

https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles

**Comment by hvnsweeting** [comment]
The change looks good, could you help revert all the format back to old style then we can merge it. Thanks

**Comment by dongocanh96** [comment]
@hvnsweeting I have reverted all the format back to old style, you can merge it now.

**Review by hvnsweeting — APPROVED** [review]

**Comment by hvnsweeting** [comment]
Awesome, thanks

**Pull request merged**